### DISCUSSION TITLE
GitHub Copilot Access Issue for recently Verified Student: No Free Access After 3 Weeks.

### DISCUSSION THREAD

**Discussion by KanaPei** [community/community#112363] [Copilot Conversations] [answered]
### Select Topic Area

Question

### Body

Hello, I verified my student status three weeks ago, but I still don’t have access to GitHub Copilot. I’ve seen cases where people had to wait a week or re-verify their student status after a long gap, but that shouldn’t apply to me since I recently verified. Here’s a screenshot where it suggests I need to pay instead of using the student version.
![image](https://github.com/community/community/assets/59775253/4eea9070-844f-4508-ae23-640e71cf143b)

One thing that occurred to me: Between two student courses, I used the 30-day free trial. Could that have affected my ability to use Copilot as a student? Thanks!

**Comment by lyuan77** [comment]
I've encountered the same issue: unable to use it and also unable to purchase.

Before applying for the student discount, I had activated a one-month trial subscription of Copilot. After obtaining the student discount, I canceled the trial subscription. However, today, it informed me that my subscription has expired and requested payment. Despite my student benefits indicating that I can use Copilot for free, in reality, both in IDEA and VSCode, I am still being notified: "You do not have access to Copilot."

**Comment by allenhung1025** [comment]
Same Issue here! I have verified for student account, but in the page for copliot, I am still required to pay for usage of copliot.
<img width="1286" alt="Screenshot 2024-03-21 at 9 46 17 AM" src="https://github.com/community/community/assets/22190213/8f89056d-ea9c-44e7-9aa3-232a88330773">

**Comment by allenhung1025** [comment, marked as answer]
@KanaPei  You can click buy, and it would show you the free offer of copilot.

**Reply by KanaPei** [reply]
That seems to be it. I wasn't expecting that, as simple as it was ! Thanks.

**Reply by ghost** [reply]
I doesn't show me the free version however i came until this part.
![copilot](https://github.com/community/community/assets/164091276/30bbae6c-c486-48f0-897f-a24aa5b20e7d)

**Reply by dr1np** [reply]
I was verified for student pack last year and is still verified now. I use copilot regularly, but recently, the plugin told me I have no access for copilot. I clicked the buy button u mentioned but the server sent a 302 redirect to 
https://github.com/settings/copilot, so I have no idea how to gain my access back.
![image](https://github.com/community/community/assets/88613370/a525dcd3-d938-46bd-8f33-4e249867842d)
